packages/cli/src/Init.ts

```diff
@@ -362,10 +366,11 @@ export class Init implements Command {
         console.log(
           `Successfully authenticated as ${bold(
             authenticationResult.email,
-          )}. Let's set up your Prisma Postgres database!`,
+          )}. Let's set up your Prisma Postgres® database!`,
         )
       }
 
+      console.log("Let's set up your Prisma Postgres® database!")
```

Comment:
Does this show the phrase twice if there were no saved credentials?

Reply:
Very good catch! It does. Will push a fix.